multirom/multirom.cpp

```diff
@@ -177,8 +182,8 @@ bool MultiROM::setRomsPath(std::string loc)
 		return false;
 	}
 
-	m_curr_roms_path = "/mnt/multirom-"TARGET_DEVICE"/";
-	mkdir("/mnt/multirom-"TARGET_DEVICE"/", 0777);
+    m_curr_roms_path = "/mnt/multirom-"  + (string)TARGET_DEVICE + "/";
+    mkdir(m_curr_roms_path.c_str(), 0777);
```

Comment:
Your editor replaces tab indentation with spaces. This file is using tabs, so please change that.